python/tk_multi_workfiles/actions/action.py

```diff
@@ -1,44 +1,85 @@
 # Copyright (c) 2015 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 """
+Menu actions.
 """
 
 import sgtk
-from sgtk import TankError
-from sgtk.platform.qt import QtGui, QtCore
 
-class Action(object):
+
+class ActionBase(object):
     """
+    Base class for Actions.
     """
     def __init__(self, label):
         """
+        Constructor.
+
+        :param label: Name of the action.
         """
         self._app = sgtk.platform.current_bundle()
         self._label = label
-        
+
     @property
     def label(self):
+        """
```

Comment:
Is the `label` used outside of the `Action` class? Do the `SeparatorAction` or `ActionGroup` classes use the label? Just wondering if it would make sense to move it into the `Action` class.